portugues/titanic/pular/boiar/computador.md~

```diff
@@ -0,0 +1,3 @@
+As pessoas não entedem muito bem como um computador chegou a costa em cima de um pedaco de madeira ainda funcionando. Como esse é o computador empresarial que tinha sido cedido a voce pela empresa ele é devolvido ao seu chefe. Ao ver os arquivos, ele nota que voce realmente trabalhou no que ele pediu. Assim que ele roda o seu código ele vê que está cheio de práticas ruins e não funciona muito bem. Você é demitido pós-mortem por causa de um aplicativo mal feito.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.